src/lib/blueprint.ts

```diff
@@ -117,14 +121,327 @@ interface ObjectProperty extends BaseProperty {
 
 type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
 
+type LowercaseMethod = Lowercase<Method>
+
 export interface TypesModule {
-  openapi: Openapi
+  openapi: OpenAPI
 }
 
 export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
+  const isFakeData = openapi.info.title === 'Foo'
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
   return {
     name: openapi.info.title,
-    routes: [],
-    resources: {},
+    routes: createRoutes(openapi.paths, isFakeData, targetPath),
+    resources: createResources(
+      openapi.components.schemas,
+      isFakeData,
+      targetSchema,
+    ),
+  }
+}
+
+const createRoutes = (
+  paths: OpenAPIPaths,
+  isFakeData: boolean,
+  targetPath: string,
+): Route[] => {
+  return Object.entries(paths)
+    .filter(([path]) => isFakeData || path === targetPath)
+    .map(([path, pathItem]) => createRoute(path, pathItem))
+}
+
+const createRoute = (path: string, pathItem: OpenAPIPathItem): Route => {
+  const pathParts = path.split('/')
+  const routePath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    path: routePath,
+    namespace: { path: `/${pathParts[1]}` },
```

Comment:
```suggestion
    // TODO: Add namespace detection for routes containing no endpoints.
    namespace: { path: `/${pathParts[1]}` },
```

Reply:
Do you want to leave that as a TODO for another PR or should we implement this now?

Reply:
Just thought to add the comment since it looks like it's implemented but really the logic is temporary